tools/clang/test/HLSLFileCheck/hlsl/functions/arguments/inout_trunc.hlsl

```diff
@@ -0,0 +1,72 @@
+// RUN: %dxc -E CrashMain -T ps_6_0 %s | FileCheck %s -check-prefix=CHK_CRASH
+// RUN: %dxc -E WarnMain -T ps_6_0 %s | FileCheck %s -check-prefix=CHK_WARN
+
+// Test for crashes that resulted when a scalar is provided to an outvar
+// and that the new warning is produced.
+
+// CHK_WARN: warning: implicit truncation of vector type
+// CHK_WARN: warning: implicit truncation of vector type
+// CHK_WARN: warning: implicit truncation of vector type
+// CHK_WARN: warning: implicit truncation of vector type
+// CHK_WARN: warning: implicit truncation of vector type
+// CHK_WARN: warning: implicit truncation of vector type
+// CHK_WARN-NOT: warning: implicit truncation of vector type
+// CHK_CRASH: CrashMain
```

Comment:
So is a crash still expected?  I thought this sounded like this change should fix the crash.

Reply:
Never mind, I see this was checking that it didn't crash.

Reply:
Will rename all to "nocrash" to clarify as discussed.